public/views/backup.html

```diff
@@ -89,7 +89,7 @@ <h5 class="text-center" translate>Write your wallet seed</h5>
         <div class="line-t p10 size-10 text-gray text-center" ng-show="wordsC.mnemonicHasPassphrase">
             <i class="fi-alert"></i>
             <span translate>
-              This seed was created with a passphrase. To recover this wallet both the mnemonic and passphrase are needed.
+              This recovery phrase was created with a passphrase. To recover this wallet both the mnemonic and passphrase are needed.
```

Comment:
Calling the seed "recovery phrase" makes the term "passphrase" somewhat confusing... I don't see how we can improve the wordings. Also "mnemonic" is mentioned here out of the blue...